jerry-core/ecma/builtin-objects/ecma-builtin-arraybuffer.c

```diff
@@ -93,16 +108,37 @@ ecma_builtin_arraybuffer_dispatch_construct (const ecma_value_t *arguments_list_
 } /* ecma_builtin_arraybuffer_dispatch_construct */
 
 /**
- * 24.1.3.3 get ArrayBuffer [ @@species ] accessor
+ * Dispatcher of the built-in's routines
  *
- * @return ecma_value
- *         returned value must be freed with ecma_free_value
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
  */
 ecma_value_t
-ecma_builtin_arraybuffer_species_get (ecma_value_t this_value) /**< This Value */
+ecma_builtin_arraybuffer_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine identifier */
+                                           ecma_value_t this_arg, /**< 'this' argument value */
+                                           const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                  *   passed to routine */
+                                           uint32_t arguments_number) /**< length of arguments' list */
 {
-  return ecma_copy_value (this_value);
-} /* ecma_builtin_arraybuffer_species_get */
+  JERRY_UNUSED (arguments_number);
+
+  switch (builtin_routine_id)
+  {
+    case ECMA_BUILTIN_ARRAYBUFFER_OBJECT_IS_VIEW:
+    {
+      ecma_value_t argument = arguments_number > 0 ? arguments_list_p[0] : ECMA_VALUE_UNDEFINED;
+      return ecma_builtin_arraybuffer_object_is_view (this_arg, argument);
```

Comment:
passing this_args for  `ecma_builtin_arraybuffer_object_is_view ` is unnecessary